staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1272,6 +1385,15 @@ message FilesystemUsage {
     // This may not equal InodesCapacity - InodesAvailable because the underlying
     // filesystem may also be used for purposes other than storing images.
     UInt64Value inodes_used = 4;
+    // TODO: Unclear how the remaining fields relate to container stats. Is it filled in cAdvisor?
```

Comment:
also TODO: I have yet to actually figure out how to wire these up yet. it's not clear this should be the responsibility of CRI impl or cAdvisor

Reply:
Does this make more sense as a cumulative stat rather than associating with an ImageSpec for a container?